### PR TITLE
Add link to documentation for approved tags and use cases

### PR DESCRIPTION
**Pull request by kgal-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
When `pack_metadata.json` includes tags and use cases that are not supported, the error message doesn't provide enough information about what tags and use cases are supported.

For example, in this [build failure](https://app.circleci.com/pipelines/github/demisto/content/177356/workflows/a9dafdda-f357-4610-8099-aecc35f4e0f9/jobs/543918?invite=true&invite=true#step-108-148) we see the current error message:

```
[ERROR]: Packs/BreachRx/pack_metadata.json: [PA120] - The pack metadata contains non approved tags: privacy, breachrx
```

The PR adds a link to the xsoar.pan.dev section where list of approved tags and use cases are listed.

### CODE REVIEW

**Review by dorschw — APPROVED** [review]

**Comment by kgal-pan** [comment]
Mention https://github.com/demisto/content/pull/20121.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Fixed an issue where the suggestion in case of `IF113` included uppercase letters for the `cliName` parameter.
 * Added new validation to the **validate** command to fail and list all the file paths of files that are using a deprecated integration command / script / playbook.
 * **validate** will no longer fail on playbooks calling subplaybooks that have a higher `fromVersion` value, if  calling the subplaybook has `skipifunavailable=True`.
+* Added link to documentation for approved use cases and tags.
```
Comment (dorschw) [inline CHANGELOG.md:11]:
```suggestion
* Added link to documentation for error messages regarding use cases and tags.
```

**Comment by kgal-pan** [comment]
@dorschw - Looks like there are [some build failures](https://app.circleci.com/pipelines/github/demisto/demisto-sdk/17616/workflows/65fe7c2d-02dc-43ae-9ca7-47adf2c8df9b/jobs/116216) relating to the `typed-ast` package:

```
ERROR: Wheel 'typed-ast' located at /path/to/wheel.whl is invalid.
```

Any idea how to fix this? BTW, I reviewed the [`typed-ast` package release notes for latest (`1.5.4`)](https://pypi.org/project/typed-ast/1.5.4/) and noticed that they recommend using `ast` for parsing code later than Python 3.8:

>Note: The `ast` module of Python 3.8+ supports all features of `typed_ast`. `typed_ast` does not support parsing code that uses syntax introduced in Python 3.8 onwards. We recommend using ast on Python 3.8 or above.

**Comment by dorschw** [comment]
> @dorschw - Looks like there are [some build failures](https://app.circleci.com/pipelines/github/demisto/demisto-sdk/17616/workflows/65fe7c2d-02dc-43ae-9ca7-47adf2c8df9b/jobs/116216) relating to the `typed-ast` package:
> 
> ```
> ERROR: Wheel 'typed-ast' located at /path/to/wheel.whl is invalid.
> ```
> 
> Any idea how to fix this? BTW, I reviewed the [`typed-ast` package release notes for latest (`1.5.4`)](https://pypi.org/project/typed-ast/1.5.4/) and noticed that they recommend using `ast` for parsing code later than Python 3.8:
> 
> > Note: The `ast` module of Python 3.8+ supports all features of `typed_ast`. `typed_ast` does not support parsing code that uses syntax introduced in Python 3.8 onwards. We recommend using ast on Python 3.8 or above.

The failure is a glitch that happens from time to time. Re-ran it, [looks good](https://app.circleci.com/pipelines/github/demisto/demisto-sdk/17616/workflows/d8706e88-879f-43ea-8f85-e45aa6909d0c/jobs/116232) for now.

About `typed-ast` vs `ast`, nice catch. We are already using the latter, I will look into removing it.

**Pull request merged**